enterprise/utils.py

```diff
@@ -2028,6 +2028,8 @@ def enroll_subsidy_users_in_courses(enterprise_customer, subsidy_users_info, dis
                 [{ 'user_id': <lms_user_id>, 'email': <email>, 'course_run_key': <key> } ... ]
         }
     """
+    from enterprise.api_client.lms import EnrollmentApiClient  # pylint: disable=import-outside-toplevel
+    enrollment_api_client = EnrollmentApiClient()
```

Comment:
Optional: instead of instantiating it here, we could use `enrollment_client=None` argument in the `ensure_course_enrollment_is_allowed` function, and instantiate it if it doesn't exist. This seems to be a common pattern in this file.

Reply:
In this particular case, the `ensure_course_enrollment_is_allowed` is called within a for loop. I didn't want to incur the penalty of initializing the API Client every iteration.

Reply:
@tecoholic, ah, good point. I didn't notice the loop in the collapsed (unchanged) lines. All good then 👍🏻